Source/WTF/wtf/glib/GRefPtr.h

```diff
@@ -116,7 +140,10 @@ template <typename T> class GRefPtr {
     GRefPtr(HashTableDeletedValueType) : m_ptr(hashTableDeletedValue()) { }
     bool isHashTableDeletedValue() const { return m_ptr == hashTableDeletedValue(); }
 
-    T* get() const { return m_ptr; }
+    // Borrows the raw pointer from GRefPtr.
+    // The pointer is guaranteed to be valid for as long as GRefPtr holds an owning reference
+    // to that object.
+    T* /* (transfer none) */ get() const { return m_ptr; }
     T& operator*() const { return *m_ptr; }
     ALWAYS_INLINE T* operator->() const { return m_ptr; }
```

Comment:
I wonder if we could add LIFETIME_BOUND tags here even here or a new bug as this could be out of the scope of documenting

Reply:
We could. It is technically possible to do .get() and safely keep the pointer after the GRefPtr is gone, but only if you have a guarantee that there will be at least one other strong reference holder while you use it... but relying on that is already a code smell.

Let's see what code gets flagged if I add it...